### PR TITLE
Adding MetaZ v1.0a15

### PR DESCRIPTION
**Pull request by mauricerkelly**
Adding a new cask for the MetaZ video file tagger application.

### CODE REVIEW

File: Casks/metaz.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'metaz' do
+  version '1.0a15'
+  sha256 '7dcc71a917bed0d5686884c4ec198275a1f656ce09db71fa49cc73e46933820c'
+
+  url "https://github.com/griff/metaz/releases/download/v#{version}/MetaZ-#{version}.zip"
+  name 'MetaZ'
+  homepage 'http://griff.github.io/metaz/'
+  license :oss
```
Comment (vitorgalvao) [inline Casks/metaz.rb]: Looks like `:mit`.

Comment (mauricerkelly) [inline Casks/metaz.rb]: Sorry - I completely missed the fact that there were specific licenses below the generic categories! I’ve force-pushed an update to this branch (hope you don’t mind the force push?)

Comment (vitorgalvao) [inline Casks/metaz.rb]: Not at all. That’s actually the way we prefer it.

**Comment by mauricerkelly** [comment]
@vitorgalvao thanks :-)

**Pull request merged**